<ui version="4.0" >
 <class>ExpSettingsForm</class>
 <widget class="QWidget" name="ExpSettingsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Параметры эксперимента</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QRadioButton" name="rbStrainStep" >
     <property name="text" >
      <string>Эксперимент с шагом по деформации</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QRadioButton" name="rbStressStep" >
     <property name="text" >
      <string>Эксперимент с шагом по напряжению</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Число шагов нагружения</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QSpinBox" name="sbExpStepCount" >
     <property name="maximum" >
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Скорость деформации - έ</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QDoubleSpinBox" name="dsbExpLoadingSpeed" >
     <property name="suffix" >
      <string>%·cᐨ¹</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>Время нагружения</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QDoubleSpinBox" name="dsbExpLoadingTime" >
     <property name="suffix" >
      <string>с</string>
     </property>
     <property name="maximum" >
      <double>900000.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Шаг нагружения</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QDoubleSpinBox" name="dsbStressStep" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="suffix" >
      <string>МПа</string>
     </property>
     <property name="maximum" >
      <double>5000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="label_11" >
     <property name="text" >
      <string>Модуль упругости образца</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QDoubleSpinBox" name="dsbCoefficientOfElasticity" >
     <property name="suffix" >
      <string>МПа</string>
     </property>
     <property name="decimals" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <double>9999999.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="label_9" >
     <property name="text" >
      <string>Время релаксации</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" >
    <widget class="QSpinBox" name="sbExpRelaxationTime" >
     <property name="suffix" >
      <string>с</string>
     </property>
     <property name="maximum" >
      <number>6000</number>
     </property>
     <property name="value" >
      <number>60</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbStrainStep</sender>
   <signal>toggled(bool)</signal>
   <receiver>dsbExpLoadingTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>175</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel" >
     <x>266</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbStressStep</sender>
   <signal>toggled(bool)</signal>
   <receiver>dsbStressStep</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>175</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel" >
     <x>266</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
